src/UX/Identification/IdentificationFactory.php

```diff
@@ -0,0 +1,17 @@
+<?php namespace ILIAS\UX\Identification;
+
+/**
+ * Class IdentificationFactory
+ *
+ * @author Fabian Schmid <fs@studer-raimann.ch>
+ */
+class IdentificationFactory {
+
+	/**
+	 * @inheritdoc
+	 */
+	public function core(\ILIAS\UX\Provider\Provider $provider): ProviderInterface {
```

Comment:
What other than core is possible? Make an example and note this as comment further down, so that devs that need to add such a new identification know where to place it.